### PR TITLE
Fixes #9836 - use provider type in host helpers

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: app/helpers/hosts_helper.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module HostsHelper
   def provider_partial_exist?(compute_resource, partial)
     return false unless compute_resource
 
-    compute_resource_name = compute_resource.provider_friendly_name.downcase
+    compute_resource_name = compute_resource.provider.downcase
     real_path = File.join(Rails.root, 'app', 'views', 'compute_resources_vms', 'form', compute_resource_name, "_#{partial}.html.erb")
```
Comment (domcleal) [inline app/helpers/hosts_helper.rb]: Side note, this doesn't look plugin-friendly, as it's referencing Rails.root rather than a search path.

**Comment by dLobatog** [comment]
@tstrachota :+1: from my side, can you take a look at @domcleal 's comment if possible? Ready to merge in my opinion.

**Comment by tstrachota** [comment]
@domcleal @eLobato it's a valid note, I'll update the patch to take plugins into consideration.

**Comment by dLobatog** [comment]
Merged as c2f60b5702320f4be3125d9fed8a30a2ec7a3994, thanks @tstrachota !

**Pull request closed without merging**